From d0518a61f12a352524832152f3ee80aea827c1ce Mon Sep 17 00:00:00 2001
From: "Jason M. Bills" <jason.m.bills@intel.com>
Date: Tue, 11 Jul 2023 09:02:20 -0700
Subject: [PATCH] Support GCC 13

Upstream has moved to GCC 13. This change allows building with GCC 13
and unblocks the upstream sync build.

Signed-off-by: Jason M. Bills <jason.m.bills@intel.com>
Upstream-Status: Pending
---
 src/peci/exception.hpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/peci/exception.hpp b/src/peci/exception.hpp
index 7cecb16..a6412b6 100644
--- a/src/peci/exception.hpp
+++ b/src/peci/exception.hpp
@@ -17,6 +17,7 @@
 
 #pragma once
 
+#include <cstdint>
 #include <iomanip>
 #include <sstream>
 #include <stdexcept>
-- 
2.25.1

